### PR TITLE
[KernelGen][Nvidia] Add fix_ operator with Triton kernel

### PR DESCRIPTION
**Pull request by bwbwzzz**
## Summary
Adds a Triton kernel for `fix_`.

## Testing
- Parametrized tests over multiple shapes and float16/float32/bfloat16 dtypes (15 cases)
- Validated against PyTorch reference on device side
- Tested on: Nvidia

## Performance
Test command: `pytest benchmark/test_fix_.py --level core` (NVIDIA H20)

### float16
| Size Detail | Torch Latency (ms) | Gems Latency (ms) | Speedup |
|---|---|---|---|
| 1073741824 | 1.078 | 1.089 | 0.989 |
| 64, 64 | 0.006 | 0.006 | 1.011 |
| 4096, 4096 | 0.023 | 0.023 | 0.985 |
| 64, 512, 512 | 0.023 | 0.023 | 1.007 |
| 1024, 1024, 1024 | 1.086 | 1.089 | 0.997 |

### float32
| Size Detail | Torch Latency (ms) | Gems Latency (ms) | Speedup |
|---|---|---|---|
| 1073741824 | 2.014 | 2.014 | 1.000 |
| 64, 64 | 0.006 | 0.006 | 1.006 |
| 4096, 4096 | 0.037 | 0.037 | 0.999 |
| 64, 512, 512 | 0.037 | 0.037 | 0.996 |
| 1024, 1024, 1024 | 2.014 | 2.012 | 1.001 |

### bfloat16
| Size Detail | Torch Latency (ms) | Gems Latency (ms) | Speedup |
|---|---|---|---|
| 1073741824 | 1.111 | 1.090 | 1.019 |
| 64, 64 | 0.005 | 0.006 | 0.966 |
| 4096, 4096 | 0.022 | 0.023 | 0.979 |
| 64, 512, 512 | 0.023 | 0.023 | 0.993 |
| 1024, 1024, 1024 | 1.103 | 1.094 | 1.008 |

**Arithmetic Mean Speedup: 0.997x**

## Multi-backend Testing
| Backend | Accuracy Test | Benchmark | Speedup (mean) | Notes |
|---|---|---|---|---|
| Nvidia (H20) | PASS | PASS (15 cases, --level core) | 0.997 | Primary |
| Tianshu | — | — | — | — |
| Muxi | — | — | — | — |
| Ascend | — | — | — | — |
| Hygon | — | — | — | — |

## Files Changed
- `src/flag_gems/ops/fix_.py`: Triton kernel implementation
- `tests/test_fix_.py`: Accuracy test
- `benchmark/test_fix_.py`: Performance benchmark
- `src/flag_gems/ops/__init__.py`: Register import and `__all__`
- `src/flag_gems/__init__.py`: Register to `_FULL_CONFIG`
- `conf/operators.yaml`: Add operator entry (kind: Math, stage: alpha 5.4)

### CODE REVIEW

File: benchmark/test_fix_.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import pytest
+import torch
+
+import flag_gems
+
+from . import base, consts
+
+
+@pytest.mark.fix_
+@pytest.mark.skipif(
```
Comment (bwbwzzz) [inline benchmark/test_fix_.py:10]: ？？？？？？
